stylesheet/styles.css

```diff
@@ -0,0 +1,276 @@
+* {
+  box-sizing: border-box;
+}
+body {
+  margin: 0px;
+  padding: 0px;
+}
+/* Header */
+header {
+  background-color: black;
+  color: white;
+  background-image: url('https://boone824.github.io/adopt-a-dog-winter2020/images/header-image.jpg');
+}
+#brand-container {
```

Comment:
keep all id's for js reasons, just make a class, it's ok if you only use it once.